hspy_bundle/NSIS_installer_script.nsi

```diff
@@ -307,7 +307,7 @@ SectionIn RO
 	File /r "${WINPYTHON_PATH}\*"
 	${If} $InstMode = 2
 	; Create right-click context menu entries for Hyperspy Here
-		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" install_hyperspy_here & exit"'
+		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" jupyter-context_menu-add & exit"'

```

Comment:
I agree that those terminals are annoying but, on the other hand, they can be considered a feature because they allow you to kill the notebook server. Currently there is no way to kill it from the UI. What do you think?